staging/src/k8s.io/apiextensions-apiserver/test/integration/testserver/start.go

```diff
@@ -58,7 +59,7 @@ func DefaultServerConfig() (*extensionsapiserver.Config, error) {
 	if err := options.RecommendedOptions.SecureServing.MaybeDefaultWithSelfSignedCerts("localhost", nil, []net.IP{net.ParseIP("127.0.0.1")}); err != nil {
 		return nil, fmt.Errorf("error creating self-signed certificates: %v", err)
 	}
-	if err := options.RecommendedOptions.ApplyTo(genericConfig); err != nil {
+	if err := options.RecommendedOptions.ApplyTo(genericConfig, extensionsapiserver.Scheme, nil); err != nil {
```

Comment:
Since earlier you set `options.RecommendedOptions.Admission` to `nil`, I guess you can use `nil` instead of `extensionsapiserver.Scheme` here, right?

Reply:
sure,for this test case, scheme is not use.

Reply:
@caesarxuchao @sttts said https://github.com/kubernetes/kubernetes/pull/56660#discussion_r154322859